.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.17.7
```

Comment:
Can you please remove this file.

Reply:
Have you ever used asdf ... this is a file for asdf to define the sdk's you are using. 
I would recomend keeping this as its useful for anyone using this library and does not hurt anyone.

Reply:
tbh ... its very useful for golang too.
The workflows can check the compatibility (we do this in autoscaler where we use the mod file to determine the go version and update the asdf file) but the asdf sets the golang that is currently been used for developement. 
Its obviously a personal choice for ease of use so Ill remove it as requested.